version.txt

```diff
@@ -0,0 +1 @@
+v0.1.0
```

Comment:
do we want to start at 0.0.1?

Reply:
if I remember well the correct way to name versions is to start from 0.1.0 (see here https://semver.org/)

Reply:
ah indeed.

I stand corrected.